setup.cfg

```
@@ -28,11 +28,11 @@ packages = find:
 where =
     - .
     - tests
-python_requires = >=3.8
+python_requires = >=3.10
```

```suggestion
python_requires = >=3.9
```

I would opt to include 3.9, as it will be supported almost till end of 2025 --- I think it has little downside, and enables people with older conda/docker setups to install cebra without additional changes/work.

Thoughts?